openwisp_radius/api/views.py

```diff
@@ -346,45 +346,25 @@ def validate_membership(self, user):
             raise serializers.ValidationError({'non_field_errors': [message]})
 
 
-class BatchView(DispatchOrgMixin, CreateAPIView):
-    authentication_classes = (BearerAuthentication,)
+class BatchView(CreateAPIView):
+    authentication_classes = (BearerAuthentication, SessionAuthentication)
```

Comment:
thanks, adding `SessionAuthentication` brought forth more issues, I solved them, I'll use Web API interface in the future! :smile:

Reply:
That's the way to go Ajay :+1: